bedrock/mozorg/templates/mozorg/about/forums/forums.html

```diff
@@ -93,90 +93,6 @@
 
 <!--END_LIST-->
 
-<section id="mozilla-labs" class="forum-group">
```

Comment:
There's a link to this ID in the "Categories" list at the top of the page. Should remove that.